install/optional/app-dropbox.sh

```diff
@@ -1,2 +1,7 @@
-sudo apt install -y nautilus-dropbox
-gtk-launch dropbox.desktop
+if [[ $ARCH == "amd64" ]]; then
+  sudo apt install -y nautilus-dropbox
+  gtk-launch dropbox.desktop
+elif [[ $ARCH == "arm64" ]]; then
+  #https://packages.ubuntu.com/noble/nautilus-dropbox
+  echo "Dropbox is not supported on arm64 architecture."
```

Comment:
It's not packaged, but it supported: https://github.com/dropbox/nautilus-dropbox?tab=readme-ov-file#building-dropbox-nautilus-extension

Maestral is an alternative, too: https://maestral.app

Reply:
idk building from source seems kinda sketchy? we could do that but ehh...
Maestral seems nice, and is perhaps something @dhh would like also, perhaps as just a replacement for dropbox?